android/build.gradle

```diff
@@ -23,12 +23,12 @@ apply plugin: 'com.android.library'
 
 android {
     namespace 'com.example.launchexternalapp'
-    compileSdkVersion 28
+    compileSdkVersion 31
 
     defaultConfig {
-        minSdkVersion 16
```

Comment:
I don't think the `minSdkVersion` needs to be changed for this package to compile on Flutter 3.24.0. You can leave it at 16.

Reply:
LGTM! Thanks for creating this PR